src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml

```
@@ -0,0 +1,62 @@
+setLeader: Test your knowledge of pseudo classes
+height: unset
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: |
+      Pseudo-classes select elements based on internal browser state, while
+      Pseudo-elements select browser owned parts of elements?
```

```suggestion
      Pseudo-classes act as if a class has been dynamically applied to an element, while pseudo-elements act on an element itself?
```

sounds good to me @rachelandrew 👍🏻